.github/workflows/rust.yml

```diff
@@ -79,7 +79,6 @@ jobs:
         with:
           toolchain: ${{ matrix.toolchain }}
           working-directory: ${{ matrix.directory }}
-          components: rustfmt,clippy
```

Comment:
So why do we not require these any more?

Reply:
We rely on `rust-toolchain.toml` for the required components, so the default toolchain will install them automatically. If we require another toolchain (e.g. `stable` for `error-stack`) these components are implied by `rustup set profile default`.

Reply:
This enforces that the `rust-toolchain.toml` has the correct components specified and even fails to update to a toolchain without those components.